packages/mdc-image-list/README.md

```diff
@@ -0,0 +1,138 @@
+<!--docs:
+title: "Image List"
+layout: detail
+section: components
+excerpt: "An RTL-aware Material Design image list component."
+iconId: card
+path: /catalog/image-lists/
+-->
+
+# Image List
+
+<!--<div class="article__asset">
+  <a class="article__asset-link"
+     href="https://material-components-web.appspot.com/image-list.html">
+    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/image-list.png" width="320" alt="Image list screenshot">
+  </a>
+</div>-->
+
+MDC Image List provides a RTL-aware Material Design image list component, representing an evolution of the
+[Material Design Grid List spec](https://material.io/guidelines/components/grid-lists.html).
+An Image List consists of several items, each containing an image and optionally supporting content (i.e. a text label).
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/grid-lists.html">Material Design guidelines: Grid lists</a>
+  </li>
+  <li class="icon-list-item icon-list-item--link">
+    <a href="https://material-components-web.appspot.com/image-list.html">Demo</a>
+  </li>
+</ul>
+
+## Installation
+
+```
+npm install --save @material/image-list
+```
+
+## Basic Usage
+
+### HTML Structure
+
+The HTML structure for a Standard Image List is as follows:
+
+```html
+<ul class="mdc-image-list my-image-list">
+  <li class="mdc-image-list__item">
+    <div class="mdc-image-list__image-aspect-container">
+      <img class="mdc-image-list__image" src="...">
+    </div>
+    <div class="mdc-image-list__supporting">
+      <span class="mdc-image-list__label">Text label</span>
+    </div>
+  </li>
+  ...
+</ul>
+```
+
+### Styles
+
+The structure above would be combined with an invocation of the `mdc-image-list-standard-columns` mixin, to establish how many
+columns should be displayed per line:
+
+```scss
+.my-image-list {
+  @include mdc-image-list-standard-columns(5);
+}
+```
+
+Images in a Standard Image list are constrained to 1:1 aspect ratio by default; this can be overridden using the
+`mdc-image-list-aspect` mixin documented below.
+
+## Style Customization
+
+### CSS Classes
+
+CSS Class | Description
+--- | ---
+`mdc-image-list` | Mandatory. Indicates the root Image List element.
+`mdc-image-list__item` | Mandatory. Indicates each item in an Image List.
+`mdc-image-list__image-aspect-container` | Optional. Parent of each item's image element, responsible for constraining aspect ratio. This element may be omitted entirely if images are alreadysized to the correct aspect ratio.
+`mdc-image-list__image` | Mandatory. Indicates the image element in each item.
+`mdc-image-list__supporting` | Optional. Indicates the area within each item containing the supporting text label, if the Image List contains text labels.
+`mdc-image-list__label` | Optional. Indicates the text label in each item, if the Image List contains text labels.
```

Comment:
Add row for `mdc-image-list--protected`.

Also I think we should either rename it to `mdc-image-list--protected-text`, or rename it to `mdc-image-list__supporting--protected` and apply it on the `mdc-image-list__supporting` element.

Reply:
Haha, whoops, yeah. I ended up realizing `protected` was missing and adding it before I saw this comment, which probably came in around the same time.